src/nvim/ex_docmd.c

```diff
@@ -6570,6 +6570,10 @@ do_exedit (
               )
              || *eap->arg != NUL
              ) {
+    // :e is a no-op in terminal buffers
```

Comment:
I think we should write out full commands in comments:

    // :edit ...

I will update this before merging.